tests/Unit/AcceleratorViewCopy/common.h

```diff
@@ -11,6 +11,12 @@
     printf ("error: TEST FAILED\n%s", KNRM );\
     abort();
 
+#define HostToDeviceCopyTest 0x1
+#define DeviceToDeviceCopyTest 0x2
+#define DeviceToHostCopyTest 0x4
+#define HostToDeviceAsyncCopyTest 0x18
+#define DeviceToHostAsyncCopyTest 0xFF
+
```

Comment:
I think you've actually introduced 2 bugs to these bit mask.  If you refer to line 71 and line 80 in the original avstress_0xFF.cpp, HostToDeviceAsyncCopyTest should be 0x8 and DeviceToHostAsyncCopytTest should be 0x10.

The default RUNMASK should be 0x1F to enable all the tests